tests/baselines/reference/user/puppeteer.log

```diff
@@ -26,6 +26,9 @@ lib/FrameManager.js(127,15): error TS2503: Cannot find namespace 'Protocol'.
 lib/FrameManager.js(685,57): error TS2345: Argument of type 'string | number | Function' is not assignable to parameter of type 'number'.
   Type 'string' is not assignable to type 'number'.
 lib/FrameManager.js(773,15): error TS2503: Cannot find namespace 'Protocol'.
+lib/Launcher.js(120,11): error TS2322: Type 'string[]' is not assignable to type 'StdioOptions'.
```

Comment:
`@types/node` got stricter in DefinitelyTyped/DefinitelyTyped#29151. Not sure this is a good thing.